babyshop_app/babyshop/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
```suggestion
ALLOWED_HOSTS = ['localhost', '127.0.0.1']
```

We dont want to allow all hosts in a production setting. Please describe that the user has to add his ip if he wants to deploy this repo to a machine.